romana-install/roles/romana/postinstall/kubernetes/master/templates/demo.sh

```diff
@@ -49,6 +49,13 @@ function get_pods() {
 function get_pod_ip() {
     kubectl ${2:+--namespace "$2"} get pod "$1" -o json | jq -r '.status.podIP'
 }
+function delete_tenant() {
+    TENANT_ID=`mysql -s --disable-column-names -u root --password=secrete tenant --execute='select id from tenants where name='\""$1"\"`
```

Comment:
Should use $(...) instead of backticks.
Password could be `{{ stack_password }}` instead.
Carefully mix the quoting instead of escaping them:
`...name="'"$1"'"``, or use `printf` to sub a value into it.